build/rbac/keep-rbac-yaml.sh

```diff
@@ -1,9 +1,5 @@
 #!/usr/bin/env bash
 set -eEuo pipefail
-# setting locate `LC_ALL=C` because different OS do files sorting differently, 
-# so setting a common behaviour, `C` sorting order is based on the byte values,
-# Reference: https://blog.zhimingwang.org/macos-lc_collate-hunt
-LC_ALL=C
```

Comment:
I moved this to where the `sort` happens so that the complexity of the sort is all in one place together for future reference.